src/algoliasearch.helper.js

```diff
@@ -248,6 +250,55 @@ AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
   });
 };
 
+/**
+ * Start the search for answers with the parameters set in the state.
+ * This method returns a promise.
+ * @return {AlgoliaSearchHelper}
+ * @return {promise.<FacetSearchResult>} the answer results
+ */
+AlgoliaSearchHelper.prototype.searchForAnswers = function(attributesForPrediction, queryLanguages, nbHits) {
```

Comment:
these arguments will be more flexible in an object I think

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/804/commits/e1c783f9ce843d3279b12e8ce256e6f2794073ce